internal/service/databaseuser/resource_database_user.go

```diff
@@ -134,7 +134,7 @@ func (r *databaseUserRS) Schema(ctx context.Context, req resource.SchemaRequest,
 				Computed: true,
 				Default:  stringdefault.StaticString("NONE"),
 				Validators: []validator.String{
-					stringvalidator.OneOf("NONE", "IDP_GROUP"),
```

Comment:
curious why we use validation logic in this resource?

Reply:
yeah should we avoid doing this? as long as the documentation captures the enums allowed, we can avoid hardcoding them

Reply:
Removing the validation of this field. Will reach out to the team which owns the APIs in case we find endpoints which are not explicit with the validation errors

Reply:
+1 to not be afraid of changing any code we don't understand 😄